deployments/sandbox/environment

```diff
@@ -34,4 +34,4 @@ _set AZUL_ES_DOMAIN "azul-index-dev"
 # Ensure that monitoring and pushing to Grafana is enabled
 _set AZUL_ENABLE_MONITORING 1
 
-_set AZUL_DEBUG 1
+_set AZUL_DEBUG 0
```

Comment:
We don't want to disable AZUL_DEBUG for the sandbox as a whole. We want to disable it for Gitlab builds, independently of whether they run in the sandbox, dev or any other deployment.